trillian/integration/demo-script.sh

```diff
@@ -7,6 +7,15 @@ if [ $(uname) == "Darwin" ]; then
 else
   URLOPEN=xdg-open
 fi
+hash ${URLOPEN} 2>/dev/null || { echo >&2 "WARNING: ${URLOPEN} not found - browser windows will fail to open"; }
+if [[ ! -d "${GOPATH}" ]]; then
+  echo "Error: GOPATH not set"
+  exit 1
+fi
+if [[ ${PWD} -ef ${GOPATH}/src/github.com/google/certificate-transparency-go/trillian/integration ]]; then
```

Comment:
Should/could it check that ${PWD} == "${GOPATH}/src/github.com/google/certificate-transparency-go"?

Reply:
The script is OK to run from other places, just not `./trillian/integration/` because it makes a local `demo-script.cfg`